package/yast2-network.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Tue Oct  2 19:59:14 UTC 2018 - mfilka@suse.com
+
+- bnc#1109312
+  - avoid internal error replaced calls to obsolete
```

Comment:
np: the description is not clear at all, I mean, you are like joining two different sentences without any link word

Reply:
`replaced calls to obsolete LanItems::FreeDevices method avoiding an internal error`